redaxo/src/core/pages/setup.php

```diff
@@ -6,13 +6,35 @@
 
 $step = rex_request('step', 'int', 1);
 $lang = rex_request('lang', 'string');
+$func = rex_request('func', 'string');
 
 $context = new rex_context([
     'page' => 'setup',
     'lang' => $lang,
     'step' => $step,
 ]);
 
+// ---------------------------------- Global Step features
+
+$cancelSetupBtn = '';
+if (!rex_setup::isInitialSetup()) {
+    $cancelSetupBtn = '
+    <style>
+        .rex-cancel-setup {
+            position: absolute;
+            top: 17px;
+            right: 40px;
```

Comment:
gleiche positionierung wie in der whoops fehlerseite oben der 'activate safe-mode' button